### DISCUSSION TITLE
Receiving bad request when using multiple applications with OIDCPathScope

### DISCUSSION THREAD

**Discussion by Sugavanam** [OpenIDC/mod_auth_openidc#1129] [General]
Hello Team,
  I'm using the apache reverse proxy with multiple applications in the backend. I use different OIDCPathScope for each application.
  If i'm logged in the first application and in new tab ill try to log the XYZ app. 
  Since the session is already there in browser, the scope of the first application(app.abc) is shared to the XYZ also without making call to OP.
   Is there any way to pass the right scopes between the application ?
   
  **Example:**
  
 Oidcredirecturi /redirect t_uri/
  Oidcscope openid profile
 Oidcprovideruri xxxxx
 Oidc clientsecret xxxxx
 Oidcclientid xxxxx
 
<location /abc>
AuthType openid-connect
OIDCPathAuthRequestParams acr_values=L2
OIDCUnAutzAction auth
OIDCPathScope "app.abc"
Require valid-user
ProxyPass https://example.com/abc
ProxyPassReverse https://example.com/abc
</Location>

<location /xyz>
AuthType openid-connect
OIDCPathAuthRequestParams acr_values=L2
OIDCUnAutzAction auth
OIDCPathScope "app.xyz"
Require valid-user
ProxyPass https://example.com/xyz
ProxyPassReverse https://example.com/xyz
  </Location>
  
  I'm using the **client-cookie** in my application

**Comment by zandbelt** [comment]
you'll also need to apply the appropriate `Require claim` directives to make sure that the scopes are indeed existing, for the same see https://github.com/OpenIDC/mod_auth_openidc/discussions/1049

**Comment by Sugavanam** [comment]
That was the exact problem .Thanks for the suggestion
